packages/hawtio/src/plugins/shared/jolokia-service.test.ts

```diff
@@ -23,12 +24,14 @@ describe('JolokiaService', () => {
 
   test('getJolokia - optimised', async () => {
     jolokiaService.getJolokiaUrl = jest.fn(async () => '/test')
-    Jolokia.prototype.list = jest.fn((path?: string | string[] | ListRequestOptions, opts?: ListRequestOptions) => {
+    Jolokia.prototype.list = jest.fn((path?: string | string[] | RequestOptions, opts?: SimpleRequestOptions) => {
       if (typeof path !== 'string') throw new Error('String is expected for path')
       if (!opts) throw new Error('No options set')
-      if (!opts.success) throw new Error('No success option set')
+      if (!opts.success) {
+        throw new Error('No success option set')
+      }
 
-      opts.success({
+      ;(opts.success! as SimpleResponseCallback)({
```

Comment:
this `;` at the beginning doesn't look cool. Isn't it something we can remove?

Reply:
This was added by `yarn format:fix` somehow... Let me check

Reply:
checked again - I removed these semicolons and I got:
```
09:32 $ yarn format:check
Checking formatting...
[warn] packages/hawtio/src/plugins/shared/jolokia-service.test.ts
[warn] packages/hawtio/src/plugins/shared/jolokia-service.ts
[warn] packages/hawtio/src/ui/util/ExpandableText.tsx
[warn] Code style issues found in 3 files. Run Prettier with --write to fix.
```

`format:fix` got the semicolons back...

Reply:
maybe removing the type cast can remove the necessity of `;`

Reply:
I'm getting:
```
TS2349: This expression is not callable.
  Not all constituents of type "ignore" | ErrorCallback are callable.
    Type "ignore" has no call signatures.
```
without casting

Reply:
ok, I can check that later